backend/src/db/migrations/1579387929111-merge_duplicate_location_nodes.js

```diff
@@ -0,0 +1,77 @@
+import { throwError, concat } from 'rxjs'
+import { flatMap, mergeMap, map, catchError } from 'rxjs/operators'
+import { getDriver } from '../neo4j'
+
+export const description = `
+  This migration merges duplicate :Location nodes. It became
+  necessary after we realized that we had not set up constraints for Location.id in production.
+`
+export function up(next) {
+  const driver = getDriver()
+  const rxSession = driver.rxSession()
+  rxSession
+    .beginTransaction()
+    .pipe(
+      flatMap(transaction =>
+        concat(
+          transaction
+            .run(
+              `
+                    MATCH (location:Location)
+                    RETURN location {.id}
+                  `,
+            )
+            .records()
+            .pipe(
+              map(record => {
+                const { id: locationIds } = record.get('location')
+                return { locationIds }
```

Comment:
Why is this plural?

Reply:
good question... it actually should be, really